whois-nrtm4-client/src/test/java/net/ripe/db/nrtm4/client/processor/UpdateNotificationFileProcessorTestIntegration.java

```diff
@@ -37,7 +37,7 @@ public void process_UNF_when_already_created_same_version_then_version_not_added
         updateNotificationFileProcessor.processFile();
 
         final List<NrtmClientVersionInfo> versionInfosPerSource = nrtm4ClientInfoRepository.getNrtmLastVersionInfoForUpdateNotificationFile();
-        assertThat(versionInfosPerSource.getFirst().version(), is(1L));
+        assertThat(versionInfosPerSource.getFirst().version(), is(2L));
```

Comment:
Why has the version changed?

Reply:
Update Notification File version is always the latest version of snap/delta
Here we are inserting one snap with version 1 and one delta with version 2
so the update notification file version will be 2...because the last version is the one of the delta (v2)

Reply:
first point described in the description above